src/pickup.cpp

```diff
@@ -191,8 +192,39 @@ bool pick_one_up( item_location &loc, int quantity, bool &got_water, bool &offer
     const auto wield_check = u.can_wield( newit );
     if( newit.has_owner() &&
         newit.get_owner() != g->faction_manager_ptr->get( faction_id( "your_followers" ) ) ) {
-        if( !query_yn( "Picking up this item will be considered stealing, continue?" ) ) {
-            return false;
+        if( u.get_value( "THIEF_MODE" ) ==
+            "THIEF_ASK" ) { // Has the player given input on if stealing is ok?
+            // y/n query_yn( _( "Picking up this item will be considered stealing, continue?" ) )
+            std::string answer = query_popup()
+                .context( "YESNOALWAYSNEVER" )
+                .message( "%s",
+                 _( "Picking up this item will be considered stealing, continue?" ) )
+                .option( "YES" ) // yes, steal all items in this location that is selected
+                .option( "NO" ) // no, pick up only what is free
+                .option( "ALWAYS" ) // Yes, steal all items and stop asking me this question
+                .option( "NEVER" ) // no, only grab free item and never ask me again
```

Comment:
Have you tried this with a non-default language? The options are not defined within that context in the keybindings file and should will therefor not have any default binding, nor any translation.

Reply:
I am actually trying to figure this out. If you could point me in the direction of where to create context and set bindings that would be kind. thanks.

Reply:
i think I found it in keybindings.json.  But this is new text so it wont be translated anyway. Maybe I can add that in one language and try

Reply:
Can you check the update @BevapDin and see if that is sufficient? I don't seem to be able to test with other languages, nor run `lang/update_pot.sh`

Reply:
Add a "name" entry to your new entries in "data/raw/keybindings.json". See the existing entries for "id": "YES".

The actual translation will be done by others, see doc/TRANSLATING.md